recipe/meta.yaml

```diff
@@ -191,7 +190,7 @@ outputs:
       host:
         - libblas
         - libcblas
-        - boost-cpp
+        - libboost-devel
```

Comment:
Strictly speaking the headers here would be enough from the POV of the run-export, but since this is run-depending on `fenics-libdolfin`, which brings the boost libs (and `fenics-dolfin` already has a huge number of superfluous run-deps), I'm not going to touch this one here.